dat-archive-idb-polyfill.js

```diff
@@ -0,0 +1,24 @@
+// Setup storage to indexdb
+const DefaultManager = DatArchive.DefaultManager
```

Comment:
You may want to add code here to double check that `DatArchive.DefaultManager` exists before trying to set the manager. (for when the code is loaded in beaker)